src/index.html

```diff
@@ -1,16 +1,61 @@
+<!-- #region HEAD ---------------------------->
 <!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+        name="viewport"
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta
+        http-equiv="X-UA-Compatible"
+        content="ie=edge"
```

Comment:
Please fix this in the whole file.
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/54860641/d8903479-68fe-4226-a50c-1fc978991d59)